charts/agent/templates/configmap-deployment.yaml

```diff
@@ -47,6 +47,9 @@ data:
   If these conditions are true, then we use the agent.sslCaFileName for the http_proxy.ca_certificate
 */}}
 {{- $baseSettings := .Values.sysdig.settings -}}
+{{- if and (include "agent.gke.autopilot" .) (or (not (hasKey $baseSettings "promscrape_web_address")) (eq (get $baseSettings "promscrape_web_address") "127.0.0.1:9990") ) }}
+  {{- $baseSettings := mergeOverwrite $baseSettings (dict "promscrape_web_address" "127.0.0.1:9991") -}}
+{{- end }}
```

Comment:
This should not be needed. `prometheus.enabled` is set to `false` earlier in the configuration, and promscrape will not run in that situation. 
cc @thom-sd

Reply:
nice catch, thanks!